packages/react/src/components/ProgressIndicator/ProgressIndicator.js

```diff
@@ -43,36 +43,36 @@ export const ProgressStep = ({ ...props }) => {
     }
   };
 
-  const currentSvg = current && (
-    <svg>
-      <path d="M 7, 7 m -7, 0 a 7,7 0 1,0 14,0 a 7,7 0 1,0 -14,0" />
-      <title>{description}</title>
-    </svg>
-  );
-
-  const completeSvg = complete && (
-    <CheckmarkOutline16 aria-label={description} role="img">
-      <title>{description}</title>
-    </CheckmarkOutline16>
-  );
-  const incompleteSvg = (() => {
-    if (complete) {
-      return null;
-    }
+  const SVGIcon = () => {
     if (invalid) {
       return (
         <Warning16 className={`${prefix}--progress__warning`}>
           <title>{description}</title>
         </Warning16>
       );
     }
+    if (current) {
+      return (
+        <svg>
```

Comment:
Where's this SVG asset coming from?

Reply:
this is carried over from the existing code

Reply:
@emyarod is this to say that we're unable to fix it as well in this PR? 🤔

Reply:
@joshblack this is to say unless there is an existing icon we can substitute here the net new icon design work can be done separately from this PR

Reply:
@emyarod totally fair! Just was curious where it came from and if we could substitute it with an official one.

@laurenmrice do you know if the icon for the "current" step of progress indicator is one from our library or is it cool if it's a custom one here?

Reply:
@joshblack sorry for late response, current step icon should be the radio-button--filled-16 using `interactive-04` token

Reply:
@laurenmrice just checking, is radio-button--filled the same as radio-button--checked? I don't see a radio-button--filled in the list of icons

Reply:
no the filled we used to have was a solid colored circle. It no longer exists in our current library so it will need to be custom